proto/tribune_tts.proto

```diff
@@ -48,13 +60,70 @@ message SynthesizeRequest
 // Provides information to the synthesizer that specifies how to process the request.
 message SynthesizeConfig
 {
-    // Desired sampling frequency in hertz of synthesized audio.
+    // Desired sampling frequency in hertz of synthesized audio. The value 0 means use the default Synthesizer sampling rate.
     int32 sample_rate_hertz = 1;
+
+    // Requested format of the output audio stream.
+    OutputFormat output_format = 2;
+
+    // ISO 639-1 code of the language of text to be synthesized (may be overriden by SSML tags in request text).
+    string language = 3;
```

Comment:
Does `language` should be placed directly in `SynthesizeConfig`, or `Voice` is better place? Just asking, maybe expected behaviour is that when `voice` is not defined, the default one for selected language is used ?

Reply:
In SSML we have separate tags for switching between languages (<lang>) and voices (<voice>), so perhaps placing a language directly in SynthesizeConfig is OK..